copier/vcs.py

```diff
@@ -79,6 +79,8 @@ def get_repo(url: str) -> OptStr:
 
     if url.startswith("git+"):
         url = url[4:]
+    elif url.startswith("https://") and not url.endswith(GIT_POSTFIX):
+        url = "".join(url, GIT_POSTFIX)
```

Comment:
I understand that most git-parsers will redirect to the .git; however, I wouldn't want to rely on that. For example, `git clone https://gitlab.com/gitlab-org/gitlab` will return `warning: redirecting to https://gitlab.com/gitlab-org/gitlab.git/`. 

Since the if statement on lines 72 through 77 screen out any urls that don't meet one of our valid examples, the only https:// urls reaching my elif statement are either github or gitlab urls. Then, if they don't have the .git on the end, we add it for them.

Once again, please let me know if I'm not understanding the scope of the project. The only places I found get_repo called indicated that it would return a git-parsable URL.

Reply:
Yes, this comment was in line with https://github.com/copier-org/copier/pull/677#discussion_r880814580.

However, if you prefer to keep it only for gitlab and github, this change is OK (although still not strictly needed).